app/models/katello/docker_tag.rb

```diff
@@ -26,6 +26,14 @@ class DockerTag < Katello::Model
 
     delegate :relative_path, :environment, :content_view_version, :product, :to => :repository
 
+    def associated_meta_tag
```

Comment:
I think it probably can't hurt to whip up a unit test for each of these.

Reply:
updated :)